english/zoo/zoo.md

```diff
@@ -1,2 +1,4 @@
 Recently I visited a zoo in my hometown. It was an awesome experience to be part of the
 nature. I saw a lot of tigers, elephants, deer and a whole lot of other [animals](../animal/animal.md)
+
+I was pretty thirsty. There is a lemonade stand over there. I head over and buy some [lemonade](english/drinks/lemonade.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.